uvicorn/server.py

```diff
@@ -130,7 +139,7 @@ def _share_socket(sock: socket.SocketType) -> socket.SocketType:
             uds_perms = 0o666
             if os.path.exists(config.uds):
                 uds_perms = os.stat(config.uds).st_mode
-            server = await asyncio.start_unix_server(
+            server = await asyncio.start_unix_server(  # type: ignore
```

Comment:
probably we can `assert sys.platform != 'win32'` instead

Reply:
I've tried that, and I've also tried `assert hasattr(asyncio, "start_unix_server")`. Neither of those works.

Any suggestion?

Reply:
> I'm not confident enough to just ignore now. Any other suggestion?

I'm out of ideas, did you try to ask on https://gitter.im/python/typing ?

Reply:
I've just asked: https://gitter.im/python/typing?at=60d71d56110daa37b1085b19

Let's wait. :)